### DISCUSSION TITLE
GitHub will remove the "Block User" feature

### DISCUSSION THREAD

**Discussion by XuanChen1234** [community/community#112195] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Without the "Block User" feature, he may be forced to leave the GitHub platform because users can't prevent some spamming, harassment and political attacks.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @XuanChen1234! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, are you having trouble blocking or reporting users on our platform? 

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by cehfisher** [comment]
Hi @ghostinhershell - it doesn't look like this was moved out of the Accessibility category. Could you make this change please? (cc @queenofcorgis)
